src/dlgConnectionProfiles.cpp

```diff
@@ -718,6 +718,15 @@ QString dlgConnectionProfiles::getDescription(const QString& hostUrl, const quin
                  * dar forma a tu legado, forjar maravillas olvidadas para ti -o tus aliados- y luchar por fe, gloria o dinero.
                  * -- end translation --
                  */
+    } else if (hostUrl == QStringLiteral("mud.clessidra.it")) {
+        return QStringLiteral(
+                "Clessidra is the first all italian MUD ever created! On Clessidra you may find only original Areas, all in italian! Many features make Clessidra one of the best, or the best, MUD in Italy : Advanced travel mode, fight one to one versus your friend, or enemy, The Arena and its fight, the Mortal Challenge, the intelligent MOBs and their Quest and fighting style, a random automatic mission assignament and for you and your friends you must try the advanced Clan system that allows wars and conquest. A mercenary system to help playing when few players are online, a crafting system to create special object and a graphical user interface to help newbie and expert players have a better experince. A MUD that evolves with new challenge, new rules, new skills!");
```

Comment:
Actually it makes more sense to have the Italian text here - as the people who will want to play it will be the ones who understand it.

We ought to do the same for the Spanish "Reinos de Leyenda" MUD as well...